erpnext/stock/doctype/stock_entry_type/stock_entry_type.py

```
@@ -31,6 +30,10 @@ class StockEntryType(Document):
 			"Repack",
 			"Send to Subcontractor",
 			"Disassemble",
+			"Receive from Customer",
```

Ruff (0.13.3)

Syntax error in forward annotation: Unexpected token at the end of an expression

(F722)